examples/spark/README.md

```diff
@@ -119,96 +157,104 @@ program.
+spark-worker-controller-fiivl   1/1       Running   0          6m
+spark-worker-controller-ytc7o   1/1       Running   0          6m
 
-$ kubectl logs spark-master
+$ kubectl logs spark-master-controller-5u0q5
 [...]
-15/06/26 14:15:43 INFO Master: Registering worker 10.244.2.35:46199 with 1 cores, 2.6 GB RAM
-15/06/26 14:15:55 INFO Master: Registering worker 10.244.1.15:44839 with 1 cores, 2.6 GB RAM
-15/06/26 14:15:55 INFO Master: Registering worker 10.244.0.19:60970 with 1 cores, 2.6 GB RAM
+15/10/26 18:20:14 INFO Master: Registering worker 10.244.1.13:53567 with 2 cores, 6.3 GB RAM
+15/10/26 18:20:14 INFO Master: Registering worker 10.244.2.7:46195 with 2 cores, 6.3 GB RAM
+15/10/26 18:20:14 INFO Master: Registering worker 10.244.3.8:39926 with 2 cores, 6.3 GB RAM
 ```
 
 ## Step Three: Start your Spark driver to launch jobs on your Spark cluster
 
 The Spark driver is used to launch jobs into Spark cluster. You can read more about it in
-[Spark architecture](http://spark.apache.org/docs/latest/cluster-overview.html).
+[Spark architecture](https://spark.apache.org/docs/latest/cluster-overview.html).
 
-```shell
-$ kubectl create -f examples/spark/spark-driver.json
+```console
+$ kubectl create -f examples/spark/spark-driver-controller.yaml
+replicationcontrollers/spark-driver-controller
 ```
 
 The Spark driver needs the Master service to be running.
 
 ### Check to see if the driver is running
 
-```shell
-$ kubectl get pods
-NAME                                           READY     REASON    RESTARTS   AGE
-[...]
-spark-master                                    1/1       Running   0          14m
-spark-driver                                    1/1       Running   0          10m
+```console
+$ kubectl get pods -lcomponent=spark-driver
+NAME                            READY     STATUS    RESTARTS   AGE
+spark-driver-controller-vwb9c   1/1       Running   0          1m
 ```
 
 ## Step Four: Do something with the cluster
 
-Use the kubectl exec to connect to Spark driver
+Use the kubectl exec to connect to Spark driver and run a pipeline.
 
-```
-$ kubectl exec spark-driver -it bash
-root@spark-driver:/#
-root@spark-driver:/# pyspark
+```console
+$ kubectl exec spark-driver-controller-vwb9c -it pyspark
 Python 2.7.9 (default, Mar  1 2015, 12:57:24)
 [GCC 4.9.2] on linux2
 Type "help", "copyright", "credits" or "license" for more information.
-15/06/26 14:25:28 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
 Welcome to
       ____              __
      / __/__  ___ _____/ /__
     _\ \/ _ \/ _ `/ __/  '_/
-   /__ / .__/\_,_/_/ /_/\_\   version 1.4.0
+   /__ / .__/\_,_/_/ /_/\_\   version 1.5.1
       /_/
+
 Using Python version 2.7.9 (default, Mar  1 2015 12:57:24)
 SparkContext available as sc, HiveContext available as sqlContext.
->>> import socket
->>> sc.parallelize(range(1000)).map(lambda x:socket.gethostname()).distinct().collect()
```

Comment:
@zmerlynn i'd request that you put this parallelize line back into the example. it serves a double purpose, providing output that can be compared to the get pods output above as a diagnostic.